### PR TITLE
fix(safe-claiming-app): loading state of unredeemed tokens

### PR DESCRIPTION
**Pull request by schmanu**
## What it solves
Solves loading issue with current deployment where unredeemed allocations caused the NoAirdrop screen to show.

### CODE REVIEW

File: apps/safe-claiming-app/src/hooks/useSafeTokenAllocation.ts [hunk 1]
```diff
@@ -141,11 +141,21 @@ const fetchTokenBalance = async (
...
+const computeVotingPower = (allocationData: Vesting[], balance: string) => {
+  const tokensInVesting = allocationData.reduce(
+    (acc, data) =>
+      data.isExpired ? acc : acc.add(data.amount).sub(data.amountClaimed),
+    BigNumber.from(0)
+  )
+
+  // add balance
+  return tokensInVesting.add(balance || "0")
+}
```
Comment (iamacook) [inline apps/safe-claiming-app/src/hooks/useSafeTokenAllocation.ts:153]: Can you add a quick test for this?

File: apps/safe-claiming-app/src/widgets/ClaimingWidget.tsx [hunk 2]
```diff
@@ -149,11 +149,14 @@ const ClaimingWidget = () => {
         >
           <SafeIcon />
           <Typography variant="h5" color="text.primary">
-            {formatAmount(Number(ethers.utils.formatEther(votingPower)), 2)}{" "}
+            {formatAmount(
+              Number(ethers.utils.formatEther(votingPower ?? 0)),
```
Comment (iamacook) [inline apps/safe-claiming-app/src/widgets/ClaimingWidget.tsx:153]: I wouldn't default to `0`. I'd suggest we show a `Skeleton` instead.

File: apps/safe-claiming-app/src/hooks/useSafeTokenAllocation.ts (hunk 1 above)
Comment (schmanu) [reply apps/safe-claiming-app/src/hooks/useSafeTokenAllocation.ts:153]: This is tested through the `useSafeTokenAllocation.test.ts`

I just refactored it out into its own function here to have it better readable.

**Review by iamacook — APPROVED** [review]

**Review by usame-algan — APPROVED** [review]

**Comment by liliya-soroka** [comment]
Verified: check on Ethereum and Goerli with diff state of allocations

**Pull request merged**